<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuestView</class>
 <widget class="QWidget" name="GuestView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guest View</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabOrders">
    <attribute name="title">
     <string>MakeOrder</string>
    </attribute>
    <widget class="QDateEdit" name="dateBegin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2018</year>
       <month>6</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="minimumDate">
      <date>
       <year>2018</year>
       <month>6</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEnd">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2018</year>
       <month>6</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="minimumDate">
      <date>
       <year>2018</year>
       <month>6</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата заселения</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата выселения</string>
     </property>
    </widget>
    <widget class="QPushButton" name="vacantRoms">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Rooms</string>
     </property>
    </widget>
    <widget class="QComboBox" name="orderplacesCb">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Places:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_46">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="orderRoomType">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="orderRatingStartCb">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_47">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rating:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="vacantRoomsTbl">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>611</width>
       <height>231</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_48">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="orderPriceEndCb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10000</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="orderRatingEndCb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="makeOrderBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Make order</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hotel:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="hotelCb">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="HotelsTbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>861</width>
       <height>171</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>250</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hotels</string>
     </property>
    </widget>
    <widget class="QComboBox" name="orderPriceStartCb">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10000</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>50</y>
       <width>331</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>310</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit info</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Edit info about yourself, if you want:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="editLbl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_36">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Message:</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="firstN">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="firstNL_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FirstName* :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="passportTE">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="phoneTE">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="passportL_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Passport* :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="lastN">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="firstNL_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LastName : </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="secondN">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="passwordL_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Phonenumber :</string>
      </property>
     </widget>
     <widget class="QLabel" name="firstNL_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SecondName* :</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="helloLbl">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>0</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Orders</string>
    </attribute>
    <widget class="QTableWidget" name="guestOrders">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>1191</width>
       <height>371</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Your orders:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
